src/aura/HH_AutoComplete/HH_AutoCompleteController.js

```diff
@@ -0,0 +1,23 @@
+({
+    handleNewContact: function(component, event, helper) {
+        var con = {'sobjectType': 'Contact'};
+        var autocomplete = component.find('autocomplete');
+        var name = autocomplete.get('v.displayValue');
+        var list = name.split(' ');
+        if (list.length == 0) {
```

Comment:
I don't think this case is possible.  If there was just one word, it would be the only element in the array (handled in next case)